brownie/network/contract.py

```diff
@@ -881,11 +888,16 @@ def _get_tx(owner: Optional[AccountsType], args: Tuple) -> Tuple:
         owner = None
 
     # seperate contract inputs from tx dict and set default tx values
-    tx = {"from": owner, "value": 0, "gas": None, "gasPrice": None}
+    tx = {"from": owner, "value": 0, "gas": None, "gasPrice": None, "nonce": None}
     if args and isinstance(args[-1], dict):
         tx.update(args[-1])
         args = args[:-1]
-        for key, target in [("amount", "value"), ("gas_limit", "gas"), ("gas_price", "gasPrice")]:
+        for key, target in [
+            ("amount", "value"),
+            ("gas_limit", "gas"),
+            ("gas_price", "gasPrice"),
+            ("nonce", "nonce"),
```

Comment:
This logic is translating field names from those used by `brownie` into those expected by `web3py`. Because we don't need to translate the `nonce` key, it shouldn't need to be included here.

Unrelated to your PR.. this function could also be cleaned up a lot.

Reply:
Good catch, I missed the update. As long as both the original and translated variables are needed in the dict i think this fct is fine.